css/main.css

```diff
@@ -75,22 +75,26 @@ body > * {
 
 .navbar-brand > img {
 	max-height: 22px;
-	opacity: 0.5;
+	opacity: 0.7;
 }
 
 .navbar-brand:hover > img {
 	opacity: 1;
 }
 
+.nav-link {
+	color: rgba(255, 255, 255, .7)
```

Comment:
Is there a reason this in RGB vs everything else that is in hex?

Reply:
good call, will fix